tetris.js

```diff
@@ -9,6 +9,16 @@ module.exports = {
 		let arrCoordinates_Y  = [];
 		let a = -1;
 		let localTetromino = []
+			
+		function checkThatTheFieldIsFree(elem){
+			for(let i of elem){
+				//undefined - if this part of the field is missing
+				if(field[i.x] == undefined || field[i.y] == undefined || field[i.y][i.x] == true){
+					return true
+				}
+			}
+			return false
+		}
 		
 		tetromino.forEach(function(elem){
```

Comment:
Rename all variables called 'elem' to something more suitable for the purpose of the each variable